INDEX

```diff
@@ -309,7 +309,7 @@
         "description": "Network infrastructure management for small networks: DNS, DHCP, router advertisement, and network boot.",
         "official": false,
         "primary_pkg": "dnsmasq"
-    },
+     },
```

Comment:
Please make to sure that we don't have formatting issues with json, this space is unnecessary here